src/Service/EntryManager.php

```diff
@@ -215,7 +219,7 @@ public function purge(User $user, Entry $entry): void
     {
         $this->dispatcher->dispatch(new EntryBeforePurgeEvent($entry, $user));
 
-        $image = $entry->image?->filePath;
+        $image = $entry->image?->getId();
```

Comment:
Same question as above, here `image` can be null as well, right?

Reply:
yes it can, but it is only used once and that is behind an `if ($image)`

Reply:
Yes ok. we are safe here.